docs/atomic-verify.1.md

```diff
@@ -13,7 +13,9 @@ IMAGE
 # DESCRIPTION
 **atomic verify** checks whether there is a newer image available and scans
 through all layers to see if any of the layers, which are base images themselves, have a new version available.
-If the tool finds an out of date image, it will report as such.
+If the tool finds an out of date image, it will report as such. **atomic verify** also allows for
+the state of a system image stored on disk to be validated, in that if the image has changed locally,
+details about those changes will be reported.
 
```

Comment:
I think we want to switch this definition to include all of the things that validate will attempt to do.  Do them by default and then add options to disable features.